src/features/user/Settings/EditProfile/EditProfile.module.scss

```diff
@@ -88,6 +121,81 @@
   flex-wrap: wrap;
 }
 
+
+.editProfileInputContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+.customiseProfileToggles {
```

Comment:
Perhaps `profileConsentSettings` would be more indicative of the purpose of this section. Review the related styles to see if they should be revised as well.